packages/backend/src/controllers/schedulerController.ts

```diff
@@ -155,4 +156,25 @@ export class SchedulerController extends Controller {
             ),
         };
     }
+
+    /**
+     * Get a generic job status
+     * This method can be used when polling from the frontend
+     * @param jobId the jobId for the status to check
+     * @param req express request
+     */
+    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @SuccessResponse('200', 'Success')
+    @Get('job/{jobId}/status')
+    @OperationId('getSchedulerJobStatus')
+    async getSchedulerStatus(
```

Comment:
New generic method for getting the status of a job , to keep polling

Reply:
🙌